JSDemos/Demos/Scheduler/CurrentTimeIndicator/description.md

```diff
@@ -1 +1,5 @@
-This example shows how to&nbsp;enable the current time indication. The switches under the Scheduler allow you to&nbsp;specify how to&nbsp;indicate the current time: by&nbsp;using a&nbsp;specific indicator, shading or&nbsp;both of&nbsp;them. You can also specify how often the Scheduler updates their position using the *&laquo;Update position in&raquo;* number box under the component.
\ No newline at end of file
+The Scheduler indicates current time if you enable the [showCurrentTimeIndicator](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#showCurrentTimeIndicator) property. Use the [indicatorUpdateInterval](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#indicatorUpdateInterval) property to specify how frequently the indicator should change its position.
+
+You can also shade the timetable up to the current time. To enable this feature, set the [shadeUntilCurrentTime](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#shadeUntilCurrentTime) property to **true**.
+
+In this example, you can use controls under the Scheduler to change the abovementioned properties at runtime.
```

Comment:
What do you mean by "controls under the Scheduler"? The developer can use any controls?

Also, please add a space between "above" and "mentioned".